docs/ref/forms/fields.txt

```diff
@@ -1626,4 +1635,15 @@ only requirements are that it implement a ``clean()`` method and that its
 ``label``, ``initial``, ``widget``, ``help_text``).
 
 You can also customize how a field will be accessed by overriding
-:meth:`~django.forms.Field.get_bound_field()`.
+:attr:`~django.forms.Field.bound_field_class` or override
+:meth:`.Field.get_bound_field()` if you need more flexibility when creating
+the ``BoundField``:
+
+.. method:: Field.get_bound_field(form, field_name)
+
+    Takes an instance of :class:`~django.forms.Form` and the name of the field.
+    The return value will be used when accessing the field in a template. Most
+    likely it will be an instance of a subclass of
+    :class:`.BoundField`.
```

Comment:
minor: Doesn't it have to be a subclass of a `BoundField`, or something that implements the same interface as it? I'm not sure we want to allude to duck typing.

Reply:
Does "most likely" not cover it? 🤔

Reply:
That's the phase I'm referring to when I say alluding to duck typing. If a person is familiar with it, they'll know they don't need to return a subclass of `BoundField`. If they aren't, they may wonder when wouldn't it return a subclass of `BoundField` and be confused when they can't find the reason why in the code.

Reply:
I added a suggestion:

```
    Takes an instance of :class:`~django.forms.Form` and the name of the field.
    The returned :class:`.BoundField` instance will be used when accessing the field 
    in a template.
```

Reply:
I'm good with either. I prefer the trimmed version, but that's my opinion.